### PR TITLE
User can see location and weather based on their current location

### PR DESCRIPTION
**Pull request by kaylawoodbury**
# [PT Story](https://www.pivotaltracker.com/story/show/172124185)
## User Story
```
As a user
In order to get further value from the news site
I would like personalisation of the content based on my current location
```
## Changes proposed in this pull request:
* Adds weather module to get weather data from a third party based on the user's navigation location
* Creates weather component to display the data retrieved from the module
* Adds weather display into header
* Adds test for weather

## What we have learned working on this feature:
* How to implement adding weather using a third party(open weather map.org) in the front end and capitalising on the user's browser navigator location
## Screenshots (Only if client side story)
Screenshot 2020-04-03 at 12 04 07

### CODE REVIEW

File: cypress/integration/userCanSeeWeather.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+describe("Weather div displays", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visit("/");
+    cy.route({
+        method: "GET",
+        url: `https://api.openweathermap.org/data/2.5/weather**`,
```
Comment (blake-futchi) [inline cypress/integration/userCanSeeWeather.js:7]: format spacing line 6-8

**Review by robin-lillqvist — COMMENTED** [review]
LGTM

**Review by blake-futchi — COMMENTED** [review]
overall looks good need to acquaint myself with this package more!

**Comment by robin-lillqvist** [comment]
MERGED!

**Pull request merged**